tests_integration_with_local_chain/markets/seer/test_seer_contracts.py

```
@@ -0,0 +1,34 @@
+        min_bond=str(int(1e18)),
+        openingTime=opening_time,
+        outcomes=["Yes", "No"],
+        market_name="test test test",
+    )
+
+
+def test_create_market(local_web3: Web3, test_keys: APIKeys) -> None:
+    factory = MarketFactory()
+    num_initial_markets = factory.market_count(web3=local_web3)
+    params = build_params()
+    tx_receipt = factory.create_categorical_market(
```

Ruff (0.8.2)

Local variable `tx_receipt` is assigned to but never used

Remove assignment to unused variable `tx_receipt`

(F841)